multitenancy-spring-boot-autoconfigure/src/test/java/org/axonframework/extensions/multitenancy/autoconfig/MultiTenancyAutoConfigurationTest.java

```diff
@@ -103,6 +103,8 @@ void multiTenancyAutoConfiguration() {
                                             .isExactlyInstanceOf(MultiTenantEventStore.class);
                          assertThat(context).getBean("multiTenantDeadLetterQueueFactory")
                                             .isInstanceOf(MultiTenantDeadLetterQueueFactory.class);
+                         assertThat(context).getBean("multiTenantEventScheduler")
+                                            .isExactlyInstanceOf(MultiTenantEventScheduler.class);
```

Comment:
Why isn't this test validating the existence of the `TenantEventSchedulerSegmentFactory`?

Reply:
in fact its missing for all other tenant segment resolvers, including this one. it should be separate test, specific for axon server. will add it